coalib/parsing/ConfParser.py

```diff
@@ -123,6 +123,12 @@ def __parse_lines(self, lines, origin):
                 if key == '':
                     continue
 
+                if key in current_section.contents and keys != []:
```

Comment:
this for loop is entered if and only if, current_keys is not empty ( hence keys ) right, ?